Support updates (#6797)
PostHog/posthog.com: contents/handbook/growth/customer-support.md

These are the ways in which customers can currently reach us:

- **Questions** - this offers users the best possible experience, as they can search [previously answered questions](/questions) that have been asked anywhere on posthog.com in our Docs!
- **In-app** - this adds extra PostHog context to Zendesk tickets and allows us to work with customers 1:1 to get the problem solved
- **Slack** - our active [PostHog Users Slack](https://posthog.com/slack) community frequently post questions, and for higher-paying (or potential higher-paying) customers, through a dedicated channel on our main company Slack
- **GitHub** - sometimes users ask about the progress of [certain issues](https://github.com/PostHog/posthog) that are important to them

### Engineers are Support Heroes

> Whilst Marcus is here in a support role to triage and escalate issues for specific teams, the direct interaction between our engineering team and our users is _hugely_ valuable, and an important part of building trust in our community is the ability for users to talk directly with the people who are actually building the product.

Reviewer: ```suggestion
Whilst our support engineer, Marcus, is here to triage and escalate issues for specific teams, the direct interaction between our engineering team and our users is _hugely_ valuable. An important part of building trust in our community is the ability for users to talk directly with the engineers at PostHog who are actually building the product.
```

> Whilst our support engineer, Marcus, is here to triage and escalate issues for specific teams, the direct interaction between our engineering team and our users is _hugely_ valuable. An important part of building trust in our community is the ability for users to talk directly with the engineers at PostHog who are actually building the product.

Reviewer 2: ```suggestion
Our Support Engineer, [Marcus](https://posthog.com/community/profiles/1036), will work directly with the according engineering team to triage and escalate certain issues, since the direct interaction between our engineering team and our users is _hugely_ valuable. An important part of building trust in our community is the ability for users to talk directly with the engineers at PostHog who are actually building the product.
```
